vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass

```diff
@@ -0,0 +1,28 @@
+// Responsive Grid
+// ========================================
+
+// Responsive grid classes from $breakpoints
```

Comment:
Maybe that is in another file? To which, should this grid and that grid be within the same file?